examples/04.MuonScan/CosmicMuons.rml

```diff
@@ -26,7 +26,7 @@
         <parameter name="saveAllEvents" value="false"/>
         <parameter name="printProgress" value="true"/>
 
-		<generator type="surface" shape="wall" position="(0,0,-50)cm" size="(30,30,0)cm" rotationAngle="0" rotationAxis="(1,0,0)">
```

Comment:
In principle the idea of this example is that the muon leaves a deposit on both, up detector and down detector. For the reconstruction of a piece that is placed in between.

The geometry is build with two detectors, up and down detectors, with a 50x50cm area, and 5cm thick.

The detectors are placed at z= +30cm and -30cm.

So a generator at -20cm would imply that the muons are generated in between detectors, and it does not reproduce cosmic muons passing through both detectors.

Perhaps the generator plane could be placed at `position="(0,0,30)cm"` it should be positive if the `angularDist direction="(0,0,-1)"` is negative.

Reply:
Yes, this is more correct. But in case the sensitive detector is the lower one, If we put the muon source at +30cm, the total number of events cannot pass through the validate condition(total entries > 850). So we need to change the validate file also.